data-model/SDP-sample-records/person_sample1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/person/1",
+  "label": "John bar Me'sam (6th c. CE)",
+  "entity_type": "person",
+  "uniform_name": "John bar Me'sam, Monk, 6th c. CE",
+  "description": "A scribe, operating at the Monastery on Mount Sinai around the 6th century.",
+  "authority_files": [],
+  "associated_dates": [{
+    "type": "floruit",
+    "date_human_readable": "6th c. CE",
+    "date_normalized": "0500/0600"
+  }],
+  "associated_entities": [{
+      "type": "place",
+      "relation": "inhabited",
```

Comment:
controlled vocabulary or LOD schema? We can likely borrow these from somewhere (HAF, Syriaca, LAWD, etc.). We could start with a core set for things like ‘author-of-work’, ‘creation’, etc.